packages/blob-storage-manager/src/index.ts

```diff
@@ -1,27 +1,40 @@
+import { prisma } from "@blobscan/db";
+
 import { BlobStorageManagerBuilder } from "./BlobStorageManagerBuilder";
 import { env } from "./env";
-import { GoogleStorage, SwarmStorage } from "./storages";
+import { GoogleStorage, PrismaStorage, SwarmStorage } from "./storages";
+
+const googleStorage =
+  env.GOOGLE_SERVICE_KEY || env.GOOGLE_STORAGE_API_ENDPOINT
+    ? new GoogleStorage({
+        bucketName: env.GOOGLE_STORAGE_BUCKET_NAME,
+        projectId: env.GOOGLE_STORAGE_PROJECT_ID,
+        serviceKey: env.GOOGLE_SERVICE_KEY,
+        apiEndpoint: env.GOOGLE_STORAGE_API_ENDPOINT,
+      })
+    : null;
+const swarmStorage =
+  env.BEE_DEBUG_ENDPOINT && env.BEE_ENDPOINT
+    ? new SwarmStorage({
+        beeDebugEndpoint: env.BEE_DEBUG_ENDPOINT,
+        beeEndpoint: env.BEE_ENDPOINT,
+      })
+    : null;
+// Fallback to prisma if no storage is available
+const prismaStorage =
+  !googleStorage && !swarmStorage ? new PrismaStorage(prisma) : null;
```

Comment:
We should give enough flexibility to configure the storages in any way, that is, instead of defining prismaStorage this way, I recommend using another environment variable so we can store to postgres and gcs.

Reply:
Something like this would be ideal:

```
STORAGE_GCS_ENABLED=true
STORAGE_SWARM_ENABLED=false
STORAGE_POSTGRES_ENABLED=true
```

Then the storages would only be initialized if they are set to true. However if none is enabled we can keep this part so we fallback to postgres